From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

garyericson committed:
Removed references to the classic portal

garyericson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 11-01-remove-classic-portal-references

garyericson committed:
Updating more articles

garyericson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 11-01-remove-classic-portal-references

garyericson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 11-01-remove-classic-portal-references

garyericson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 11-01-remove-classic-portal-references

garyericson committed:
Replacing classic portal references with new portal

garyericson committed:
Fixed a couple things

garyericson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 11-01-remove-classic-portal-references

garyericson committed:
Responding to edit review

carolinacmoravia committed:
C6756: LOCBLOCKER: Extra tabulation is causing the strings below to get blocked and unlocalized 

Hello, @mahesh-unnikrishnan,
Localization team has reported source content issue that cause localized version to have broken format compared to en-us version.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.
Many thanks in advance.

rjagiewich committed:
edit pass: two-hdinsight-articles

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-hdinsight-articles

rjagiewich committed:
edit pass: two-hdinsight-articles

rjagiewich committed:
edit pass: two-hdinsight-articles

v-jonawo committed:
Merge pull request #1 from MicrosoftDocs/master

update current branch

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-hdinsight-articles

jkwong5 committed:
removing irrelevant files

jkwong5 committed:
removing irrelevant files

rjagiewich committed:
edit pass: two-hdinsight-articles

v-viksur committed:
(ForumsTeam) Create NSGs through Classic portal

In this documentation we have mentioned that we can create NSGs through Classic portal, but there is no documentation attached/hyperlinked. To my knowledge it is not possible to create NSGs through Classic Portal. could you please look in to it and edit accordingly.

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-hdinsight-articles

rjagiewich committed:
two-hdinsight-articles

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-hdinsight-articles

rjagiewich committed:
edit pass: two-hdinsight-articles

rjagiewich committed:
edit pass: two-hdinsight-articles

paulathurman committed:
edit pass: forced-tunnel-support

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into forced-tunnel-support

paulathurman committed:
edit pass: forced-tunnel-support

v-viinde committed:
USER STORY 1130240 SaaS App Tutorial: IMAGE WORKS

carolinacmoravia committed:
M15128: Possible necessary word: set

Hello, @iainfoulds, 

Translator has reported possible source content issue. It seems that the word "set" makes the sentece clearer. 

Please ,help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

v-viinde committed:
USER STORY 1134728 SaaS App Tutorial: Dropbox for Business Tutorial Update

v-viinde committed:
USER STORY 1134826 SaaS App Tutorial: Communifire

v-viinde committed:
Updated

carolinacmoravia committed:
M16459: Possible incorrect terminology: "primary key"

Hello, @mimig1, 
Translator has reported possible source content issue.  It seems it should read "partition key" according to the context. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
M16508: Possible duplicated content

Hello, @jseb225, 
Translator has reported possible source content issue. It seems that "With ASA, you" is duplicated by mistake. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

yaronyg committed:
We fixed it

jkwong5 committed:
adding documentation and TOC

jkwong5 committed:
merging

anumjs committed:
Added reference API link for Java version

garyericson committed:
Restored TOC items

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
USER STORY 1135610 SaaS App Tutorial: MOBI

v-viinde committed:
Updated

rapatchi committed:
image change and edge browser update

rjagiewich committed:
Update hdinsight-for-vscode.md

anumjs committed:
Added links for Java version

v-dotren committed:
Noted max lifetime of a task is 7 days from job add to complete

spelluru committed:
added a link to ARM template quick start from the home page

RicksterCDN committed:
fixing date

spelluru committed:
Fixed the link

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

MikeDodaro committed:
News endpoint summary topic + add to toc.yml

MikeDodaro committed:
Fix spelling error

anumjs committed:
Added links for Java version for multishard query

anumjs committed:
Added links for Java version in credential doc

MikeDodaro committed:
Edits after review

Commit:
switched kelly to author in metadata

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix-iot-edge-metadata
sync with master

Commit:
update for UI change

MikeDodaro committed:
Line feed spacing and small additions

anumjs committed:
Added Java links for adding a shard

MikeDodaro committed:
Fix spacing and a spellng error

spelluru committed:
$ variables

anumjs committed:
Changed code snippet tags

rjagiewich committed:
Update hdinsight-for-vscode.md

MikeDodaro committed:
Trying to get paragraph spacing right

rjagiewich committed:
Update hdinsight-for-vscode.md

Commit:
reviewing edits needed

rjagiewich committed:
Update set-up-pyspark-interactive-environment.md

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into v2vprice

JeffGoldner committed:
Updates for 1711 release, correct typo

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

MikeDodaro committed:
Add carraige return

JeffGoldner committed:
fix warnings

JeffGoldner committed:
fix warnings

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into v2vprice

anumjs committed:
incorporated acrolinx suggestions

Lisaco88 committed:
Merge pull request #28899 from JamesEarle/create-logic-app

update for UI change

Commit:
Fix container repo name in docs

Commit:
acrilinx edits

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix-iot-edge-metadata
sync with master

ktoliver committed:
Merge pull request #28873 from MikeDodaro/endpoint-news-summary

News endpoint summary topic + add to toc.yml

Commit:
changed filtermodule -> filterModule in second route

anumjs committed:
Fixed code snippet tags

ktoliver committed:
Merge pull request #28868 from sanjeev3/master

Minor edits to Tutorial pages

anumjs committed:
fixed indentation

PRMerger18 committed:
Merge pull request #28909 from bradsev/telemetry-walkthru-topics-updates

reviewing edits needed

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into v2vprice

ktoliver committed:
Merge pull request #28910 from JeffGoldner/master

Update for RP release 1711, fix typo

PRMerger13 committed:
Merge pull request #28915 from JimacoMS2/fix-iot-edge-metadata

Change author metadata to Kelly for c# module and function tutorials and filtermodule -> filterModule in second route in c# tutorial

Commit:
Merge pull request #28914 from suhuruli/FixContainerRepoName

Fix container repo name in docs

garyericson committed:
Merge branch 'master' into 11-01-remove-classic-portal-references

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into v2vprice

ktoliver committed:
Merge pull request #28799 from garyericson/11-28-toc-fix

Restored TOC items

anumjs committed:
Updated code snippet

Commit:
Merge pull request #28906 from spelluru/inccopy1129

$ variables

anumjs committed:
added a missing )

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into v2vprice

Commit:
updating and addressing issue with storage topic

Commit:
fixing toc context

sanjeev3 committed:
Update api-reference.md

cherylmc committed:
format

Commit:
deleted obsolete topic and added redirect

ShawnJackson committed:
Update forced-tunnel-support.md

ShawnJackson committed:
Merge pull request #28274 from paulathurman/forced-tunnel-support

Edit pass: Forced tunnel support

PRMerger13 committed:
Merge pull request #28863 from RicksterCDN/FlashGrid

fixing date

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into v2vprice

cherylmc committed:
update

PRMerger19 committed:
Merge pull request #28911 from cherylmc/v2vprice

V2V about connecting

tweissms committed:
Fix grammer issues and clarify capacity and throughput numbers

cherylmc committed:
Update vpn-gateway-howto-point-to-site-rm-ps.md

tweissms committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into upgrade-scale-updates

cherylmc committed:
update syntax typo

tweissms committed:
Fix some acrolinx warnings.

cherylmc committed:
Update vpn-gateway-howto-point-to-site-rm-ps.md

GitHubber17 committed:
Merge pull request #28879 from anumjs/multishardquery

Added links for Java version for multishard query

yujhongmicrosoft committed:
Update TOC.md

yujhongmicrosoft committed:
Update documentation-government-gpu.md

yujhongmicrosoft committed:
Update documentation-government-howto-deploy-webandmobile.md

yujhongmicrosoft committed:
Update documentation-government-functions.md

GitHubber17 committed:
Merge pull request #26684 from garyericson/11-01-remove-classic-portal-references

Removed references to the classic portal

yujhongmicrosoft committed:
Update documentation-government-cognitiveservices.md

GitHubber17 committed:
Merge pull request #28862 from spelluru/armlink1129

added a link to ARM template quick start from the home page

yujhongmicrosoft committed:
Update documentation-government-cognitiveservices.md

Commit:
Merge pull request #28926 from cherylmc/patch-2

update typo

JasonWHowell committed:
Update quickstart-create-mysql-server-database-using-azure-cli.md

GitHubber17 committed:
Merge pull request #28886 from anumjs/credentialsEDCL

Added links for Java version in credential doc

JasonWHowell committed:
Update howto-configure-server-parameters-using-cli.md

GitHubber17 committed:
Merge pull request #28787 from anumjs/datarouting

Added reference API link for Java version

JasonWHowell committed:
Update app-service-web-create-web-app-no-h.md

JasonWHowell committed:
Update app-service-web-create-app-service-plan-no-h.md

JasonWHowell committed:
Update app-service-web-create-app-service-plan-linux-no-h.md

GitHubber17 committed:
Merge pull request #28922 from sanjeev3/master

Update api-reference.md

PRMerger20 committed:
Merge pull request #28931 from JasonWHowell/patch-1

MySQL: Update quickstart-create-mysql-server-database-using-azure-cli.md

PRMerger10 committed:
Merge pull request #28932 from JasonWHowell/patch-2

MySQL: Update howto-configure-server-parameters-using-cli.md

GitHubber17 committed:
Merge pull request #28927 from tweissms/upgrade-scale-updates

 Fix grammar issues and clarify capacity and throughput numbers

PRMerger16 committed:
Merge pull request #28933 from JasonWHowell/patch-3

Update app-service-web-create-web-app-no-h.md

Commit:
Merge pull request #28934 from JasonWHowell/patch-5

Update app-service-web-create-app-service-plan-no-h.md

JasonWHowell committed:
Update configure-deployment-user-no-h.md

PRMerger17 committed:
Merge pull request #28935 from JasonWHowell/patch-6

Update app-service-web-create-app-service-plan-linux-no-h.md

GitHubber17 committed:
Merge pull request #28921 from bradsev/misc-stale-articles

updating and addressing issue with storage topic

Commit:
SaaS StandAlone App, Acrolinx. MightyPen = genemi.

sanjeev3 committed:
Update api-reference.md

GitHubber17 committed:
Merge pull request #28936 from JasonWHowell/patch-1

Update configure-deployment-user-no-h.md

JasonWHowell committed:
Fixing broken links

JasonWHowell committed:
Fixing broken links

Commit:
Merge pull request #28942 from JasonWHowell/postgresqllinks

PostgreSQL: Fixing broken links

JasonWHowell committed:
Update tutorial-docker-python-postgresql-app.md

PRMerger10 committed:
Merge pull request #28943 from JasonWHowell/patch-1

Update tutorial-docker-python-postgresql-app.md

tamram committed:
updating ms.author field - part 1

PRMerger14 committed:
Merge pull request #28945 from tamram/tamram-1129a

updating ms.author field - part 1

Commit:
Merge pull request #28825 from rapatchi/servicefabric_jenkins_image

image change and edge browser update

Commit:
Updating to reflect the current product state

We no longer have 1 minute metrics, but all metrics

PRMerger10 committed:
Merge pull request #28951 from danimir/patch-5

Updating to reflect the current product state

PRMerger12 committed:
Merge pull request #27619 from carolinacmoravia/patch-457

C6756: LOCBLOCKER: Extra tabulation is causing the strings below to g…

PRMerger14 committed:
Merge pull request #28463 from carolinacmoravia/patch-471

M15128: Possible necessary word: set

PRMerger20 committed:
Merge pull request #28718 from carolinacmoravia/patch-487

M16459: Possible incorrect terminology: "primary key"

PRMerger12 committed:
Merge pull request #28728 from carolinacmoravia/patch-490

M16508: Possible duplicated content

YuriDio committed:
updating article

yujhongmicrosoft committed:
Update documentation-government-cognitiveservices.md

ktoliver committed:
Merge pull request #28940 from sanjeev3/master

Update api-reference.md

rjagiewich committed:
Merge pull request #28938 from MightyPen/saasdbtenancy-20171129-b2-acrolinx

SaaS StandAlone App, Acrolinx.

ktoliver committed:
Merge pull request #28941 from JasonWHowell/mysqlcats2

MySQL: Fixing broken links

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ktoliver committed:
Merge pull request #28422 from v-viinde/imageworks

USER STORY 1130240 SaaS App Tutorial: IMAGE WORKS

ktoliver committed:
Merge pull request #28673 from v-viinde/dropboxforbusiness

USER STORY 1134728 SaaS App Tutorial: Dropbox for Business Tutorial U…

ktoliver committed:
Merge pull request #28679 from v-viinde/communifire

USER STORY 1134826 SaaS App Tutorial: Communifire

rjagiewich committed:
Merge pull request #28820 from v-viinde/mobi

USER STORY 1135610 SaaS App Tutorial: MOBI

PRMerger19 committed:
Merge pull request #28842 from v-dotren/batch-max-run-time

Noted max lifetime of a task is 7 days from job add to complete

ShawnJackson committed:
Update set-up-pyspark-interactive-environment.md

PRMerger19 committed:
Merge pull request #27997 from v-viksur/patch-10

(ForumsTeam) Create NSGs through Classic portal

jkwong5 committed:
corrections to md

PRMerger15 committed:
Merge pull request #28954 from YuriDio/UpdatesNov_04

updating article

v-shils committed:
Merge pull request #27941 from v-jonawo/master

Removing Unnecessary Articles

v-alje committed:
Merge pull request #28958 from MicrosoftDocs/FromPublicRepo

From public repo

ShawnJackson committed:
Update set-up-pyspark-interactive-environment.md

PRMerger10 committed:
Merge pull request #28774 from yaronyg/minor

We fixed it

ShawnJackson committed:
Merge pull request #27829 from rjagiewich/two-hdinsight-articles

Edit pass: Two hdinsight articles

Lisaco88 committed:
Merge pull request #28930 from yujhongmicrosoft/patch-7

Update TOC.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
